buildpacks/jvm-function-invoker/build.sh

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env bash
-set -euo pipefail
-shopt -s dotglob
-
-buildpack_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
-
-pushd "${buildpack_dir}"
-
-cargo libcnb package --release
-
-mkdir target
-cp -r ../../target/buildpack/release/heroku_jvm-function-invoker/* target/
-cp package.toml target/
-
```

Comment:
I think it's a leftover from an older version of these build scripts, but the `target-cargo` entry from `.gitignore` can be removed too now I believe:
https://github.com/heroku/buildpacks-jvm/blob/c9b38104c9b7c2c1088ad8f8a473bfba6a7c4525/.gitignore#L3